CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Keyboard navigation improvements for the Insights Edit view
 - Added the internal machinery to support text network analyses in the end-to-end flow.
+- Split 'Pages' tab in settings into 'Pages' and 'Policies' in preparation of nav-bar customization.
```

Comment:
Could you try to think of a gov success person and reword so they understand what happened? E.g. What was moved from one tab to the other? Where in the Settings can they find it? The info "as prep for nav bar customization" isn't necessary.

I also think it needs to be under the "Next release" header.